### ISSUE TITLE
FR: universal way to mark a button/item/menu long-pressable

### ISSUE THREAD

**Issue by weijiuqiao** [koreader/koreader#9819]
**Does your feature request involve difficulty completing a task? Please describe.**
Might be. It should involve multiple components scattered in the codebase.

**Describe the solution you'd like**
A universal way to mark a button/item/menu able to be long-pressed to invoke extra functionality.

**Additional context**
I feel that many functions stay "hidden" in KOReader behind an invisible long-press. If there's a way to show anything is long-press enabled, that would be great.
However, the marking should be as non-intrusive as possible so it doesn't make the UI ugly. I'm thinking about some kind of dot(s), short line, border, but they don't seem suitable for all situations and might be ugly :/

Any suggestions?

**Comment by poire-z** [comment]
> might be ugly 

Yeah, probably. Those of us who are used to KOReader won't need it, so anything will add noise, unbalance, and congestion to the UI...
Also, some long-press features are a bit dev-related and are better hidden (ie. long-press on "About typography rules").

I think there is one rule that everybody should know:
**in KOReader, everything is long-press able - except stuff that isn't** :)
